Userland/Libraries/LibWeb/URL/URLSearchParams.h

```diff
@@ -28,6 +28,8 @@ class URLSearchParams : public Bindings::PlatformObject {
 
     virtual ~URLSearchParams() override;
 
+    size_t size() const { return m_list.size(); }
```

Comment:
needs spec comments:

```suggestion
    // https://url.spec.whatwg.org/#dom-urlsearchparams-size
    size_t size() const
    {
        // The size getter steps are to return this’s list’s size. 
        return m_list.size();
    }
```

I'd suggest moving it out of line for that.

Reply:
> I'd suggest moving it out of line for that.

Means moving the definition to the cpp file, instead of cramming multiple lines into the header. You don't have to credit me in the commit btw :)

Reply:
Haha sorry, thought you meant one line -> multiple. Thanks!